packages/powersync_core/analysis_options.yaml

```diff
@@ -13,6 +13,12 @@
 
 include: package:lints/recommended.yaml
 
+analyzer:
+  language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
```

Comment:
These looks like good checks to have - should we enable these on all our packages? Not the demos, but anything that we publish. Can be a separate PR.

Reply:
Agreed, I'll open a PR to follow up on this 👍